swaglyrics/__main__.py

```diff
@@ -10,63 +10,84 @@
 
 
 def main():
-	# 	print(r"""
-	#  ____                     _               _
-	# / ___|_      ____ _  __ _| |   _   _ _ __(_) ___ ___
-	# \___ \ \ /\ / / _` |/ _` | |  | | | | '__| |/ __/ __|
-	#  ___) \ V  V / (_| | (_| | |__| |_| | |  | | (__\__ \
-	# |____/ \_/\_/ \__,_|\__, |_____\__, |_|  |_|\___|___/
-	#                     |___/      |___/
-	# 	""")
-	print('Updating unsupported.txt from server.')
-	with open('unsupported.txt', 'w', encoding='utf-8') as f:
-		response = requests.get('http://aadibajpai.pythonanywhere.com/master_unsupported')
-		f.write(response.text)
-	print("Updated unsupported.txt successfully.")
+    # 	print(r"""
+    #  ____                     _               _
+    # / ___|_      ____ _  __ _| |   _   _ _ __(_) ___ ___
+    # \___ \ \ /\ / / _` |/ _` | |  | | | | '__| |/ __/ __|
+    #  ___) \ V  V / (_| | (_| | |__| |_| | |  | | (__\__ \
+    # |____/ \_/\_/ \__,_|\__, |_____\__, |_|  |_|\___|___/
+    #                     |___/      |___/
+    # 	""")
+    print('Updating unsupported.txt from server.')
+    with open('unsupported.txt', 'w', encoding='utf-8') as f:
+        response = requests.get(
+            'http://aadibajpai.pythonanywhere.com/master_unsupported')
+        f.write(response.text)
+    print("Updated unsupported.txt successfully.")
+    program = "Swaglyrics"
+    parser = argparse.ArgumentParser(
+        prog=program,
+        usage='{prog} [options]'.format(prog=program),
+        description="Get lyrics for the currently playing song on Spotify. Either --tab or --cli is required.")
 
-	parser = argparse.ArgumentParser(
-		description="Get lyrics for the currently playing song on Spotify. Either --tab or --cli is required.")
+    """
+    add_mutually_exclusive_group - makes sure either -c or -t is
+    selected but not both
+    """
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('-t', '--tab', action='store_true',
+                       help='Display lyrics in a browser tab.')
+    group.add_argument('-c', '--cli', action='store_true',
+                       help='Display lyrics in the command-line.')
+    parser.add_argument('--song', help='Enter song name', type=str)
+    parser.add_argument('--artist', help='Enter artist name', type=str)
 
-	parser.add_argument('-t', '--tab', action='store_true', help='Display lyrics in a browser tab.')
-	parser.add_argument('-c', '--cli', action='store_true', help='Display lyrics in the command-line.')
+    args = parser.parse_args()
 
-	args = parser.parse_args()
+    if args.tab:
```

Comment:
indentation 😶 
and line lengths

Reply:
I see, I used that autopep8 that's why it changed everything 🙄

Reply:
let me know when it's fixed :)